gemp-lotr/gemp-lotr-async/src/main/web/src/types/set-numbers.enum.ts

```diff
@@ -0,0 +1,72 @@
+
+export enum DecipherSets {
+    Promo = 0,
+    FellowshipOfTheRing = 1,
+    MinesOfMoria = 2,
+    RealmsOfTheElfLords = 3,
+    TheTwoTowers = 4,
+    BattleOfHelmsDeep = 5,
+    EntsOfFangorn = 6,
+    ReturnOfTheKing = 7,
+    SiegeOfGondor = 8,
+    Reflections = 9,
+    MtDoom = 10,
+    Shadows = 11,
+    BlackRider = 12,
+    Bloodlines = 13,
+    ExpandedMiddleEarth = 14,
+    Hunters = 15,
+    WraithCollection = 16,
+    RiseOfSaruman = 17,
+    TreacheryAndDeceit = 18,
+    AgesEnd = 19
+}
+
+export enum PlayersCommitteeSets {
```

Comment:
If you're curious: yes i did regret this after learning more about the way it works, but this is still fine. 

I can go back with the knowledge you dropped and replace `SetXX` with something meaningful even if it's `HobbitReserved4`

Reply:
Probably not necessary; the "reserved" notes I mentioned were more casual than the word implies.  I was more communicating where I _planned_ to put things; it's a convention in other words, and no part of the system enforces any of those intents.

Also, it's "Player's Council" rather than "Player's Committee"; that was a bit of flair based on The White Council that we figured let us distinguish ourselves from the SWCCG without changing the acronym.

Reply:
good call. Thanks!

I can remove all set numbers that are not officially "released" and we can add more when they are released. The reason I had them in the first place was jCard site determination was saying `if (set === 1 || set === 51 || set === 81)` so it seemed like they were magically linked. I'd just as soon get rid of all the "in case of future" magic code like that.